src/Components/Routes.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Quote from './Quote';
+import NavBar from './Nav';
+import Home from './Home';
+import App from './App';
+
+const Routes = () => (
+  <BrowserRouter>
+    <div className="calculator">
+      <NavBar />
+    </div>
+    <Switch>
+      <Route path="/home" component={Home} exact />
+      <Route path="/app" component={App} exact />
+      <Route path="/quote" component={Quote} exact />
```

Comment:
we need to add jest snapshots for Home and Quote.
![image](https://user-images.githubusercontent.com/31897434/128343118-cabd27ec-4829-4fd5-a2a2-653d1d01f90b.png)